UNITTESTS/stubs/connectivity/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(mbed-stubs-connectivity)
+
+target_compile_definitions(mbed-stubs-connectivity
+    PRIVATE
+        DEVICE_SERIAL=1
+        DEVICE_INTERRUPTIN=1
+        MBED_CONF_CELLULAR_USE_SMS=1
+        MBED_CONF_NSAPI_DEFAULT_CELLULAR_APN=NULL
+        MBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE=115200
+        MBED_CONF_LORA_OVER_THE_AIR_ACTIVATION=true
+        MBED_CONF_LORA_AUTOMATIC_UPLINK_MESSAGE=true
+        MBED_CONF_LORA_TX_MAX_SIZE=255
+        MDMTXD=NC
+        MDMRXD=NC
+)
+
+target_sources(mbed-stubs-connectivity
+    PRIVATE
+        aes_stub.c
+        AT_CellularContext_stub.cpp
+        AT_CellularDevice_stub.cpp
```

Comment:
Open question to other reviewer: now we have
```
UNITTESTS
  - stub
    - connectivity
    - drivers
    - platform
    - ...
```
Rather than duplicating the top-level mbed-os components, what about moving stubs into their respective components? Any preferences?

Reply:
> Open question to other reviewer: now we have
> 
> ```
> UNITTESTS
>   - stub
>     - connectivity
>     - drivers
>     - platform
>     - ...
> ```
> 
> Rather than duplicating the top-level mbed-os layout, what about moving stubs into their respective components? Any preferences?

I would also prefer it if the stubs were moved into their respective components. I don't really see the value in duplicating the mbed-os layout here either.

Reply:
As discussed, we will plan this work as part of `Refactor components for independent unit testing`